plone/app/upgrade/v60/profiles/to_isiteschema/registry.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<registry>
+    <records interface="Products.CMFPlone.interfaces.ISiteSchema"
```

Comment:
Does this really only register new fields and doesn't that wipe any user made configuration in ISiteSchema?
Anyways, It would be cleaner to just add the `webstats_head_js` field.